src/backend/mpi/swdsm.cpp

```diff
@@ -512,31 +505,29 @@ unsigned long getOffset(unsigned long addr){
 	return offset;
 }
 
-void write_back_writebuffer() {
-	unsigned long i;
-	unsigned long oldstart;
-	unsigned long idx,tag;
+void write_back_write_buffer() {
```

Comment:
Maybe this also can move to the WriteBuffer class?

Reply:
flush_write_buffer is now WriteBuffer::flush().
write_back_write_buffer is now WriteBuffer::flush_partial()
add_to_write_buffer is now WriteBuffer::add(T val)
WriteBuffer::erase(T val) is used to erase a single element.

Reply:
very nice :)